core/src/main/java/nl/nn/adapterframework/configuration/IbisContext.java

```diff
@@ -129,19 +130,21 @@ public synchronized void init(boolean reconnect) {
 
 			load();
 
-			log("startup in " + (System.currentTimeMillis() - start) + " ms");
+			long startupTime = (System.currentTimeMillis() - start);
+			log("startup in " + startupTime + " ms");
```

Comment:
Shouldn't this be like this as changed in other files in this pr
```suggestion
			log("startup in {} ms", startupTime);
```